src/endpoint/devices.ts

```diff
@@ -5,24 +5,62 @@ import { Links, Status, SuccessStatusValue } from '../types'
 import { PresentationDevicePresentation } from './presentation'
 
 
-const HEADER_OVERRIDES = {Accept: 'application/vnd.smartthings+json;v=20170916'}
+const HEADER_OVERRIDES = { Accept: 'application/vnd.smartthings+json;v=20170916' }
 
 export interface CapabilityReference {
 	id: string
 	version?: number
 	status?: CapabilityStatus
 }
 
+export interface DeviceProfileReference {
+	id?: string
+}
+
+// Included for backwards compatibility (renamed to match API docs).
```

Comment:
We should switch this to a JSDoc block and add a `@deprecated` here.

Reply:
Fixed